### PR TITLE
Nominate myself(Shelley-BaoYue) as a new maintainer

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
**What this PR does / why we need it**:

I have been active in the kubeegde community for more than two years, participated in the development of major features in recent releases, also helped review lots of codes, have shown good technical judgement in feature design/development.

I would like to apply myself(Shelley-BaoYue) as a new maintainer and make more contributions to the community.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ describes the mission, scope, and objectives of the KubeEdge Technical Steering
 | Yin Ding            | @dingyin          | Google        | <dingyin@gmail.com> |
 | Zefeng (Kevin) Wang | @kevin-wangzefeng | Huawei        | <wangzefeng@huawei.com> |
 | Zhe Gong            | @GsssC            | HarmonyCloud  | <zhiyi375@gmail.com> |
+| Yue Bao             | @Shelley-BaoYue   | Huawei        | <baoyue2@huawei.com> |
```
Comment (fisherxu) [inline MAINTAINERS.md:39]: Sort alphabetically.

**Comment by WillardHu** [comment]
+1

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Comment by fisherxu** [comment]
We've got 4/7 approval from TSC members, 2 approval from core maintainers, thank you all for the support. And congratulations to @Shelley-BaoYue 💯
/approve

**Pull request merged**